packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -206,7 +199,15 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                         {min_order_amount_display}-{max_order_amount_display} {account_currency}
                     </Table.Cell>
                     <Table.Cell className='p2p-my-ads__table-price'>
-                        {price_display} {local_currency}
+                        <div className='display-layout'>
+                            {price_display} {local_currency}
+                            {rate_type === ad_type.FLOAT && (
+                                <AdType
+                                    float_rate={rate_display}
+                                    className='p2p-my-ads__table__payment-method--label'
```

Comment:
Hmm why are we passing `className` if the `AdType` will only be used with the `p2p-my-ads__table__payment-method--label`?

Reply:
Replaced with CSS that is part of then component